core/api/views.py

```
@@ -57,4 +58,220 @@
+                    },
+                    status=status.HTTP_201_CREATED
+                )
+            return Response(
+                {
+                    'status': False,
+                    'message': 'Invalid data',
+                    'errors': serializer.errors
+                },
+                status=status.HTTP_400_BAD_REQUEST
+            )
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)